docs/blog/posts/look-at-it.md

```diff
@@ -0,0 +1,77 @@
+---
+draft: False
+date: 2023-11-26
+slug: python-caching
+tags:
+  - caching
+  - functools
+  - diskcache
+  - python
+  - rag
+authors:
+  - jxnl
+---
+
+# Improving your RAG with Question Types
+
+!!! note "This is a work in progress"
+
+    This is a work in progress I'm going to use bullet points to outline the main points of the article.
+
+- I've been consulting some and a common question is "How do I improve my RAG?"
+- Theres always the black box responses like 'lets add cohere' or 'lets change chunk size', but generic solutinos get you generic results.
+- Instead my recommendation is to simply... look at your data.
+- Once we look at the data we'll have plenty of information we need to itentify the best intervention strategy and also to figure out where we might want to specialize our model.
+
+In this blog we'll cover a range of things that can lead us into the right direction. Go over some examples of companies that can do this kind of exploration. We'll leave it open ended as to what the interventions are, but give you the tools to drill down into your data and figure out what you need to do. For example if if google learns that a large portion of queries are looking for directions or a location, they might want to build a seperate index and release a maps product rather than expecting a HTML page to be the best response.
```

Comment:
For example, if Google learns that a large portion of queries is looking for directions or a location.("if" is repeated in your draft)